src/python/s2_common.i

```diff
@@ -34,6 +36,16 @@
 %apply std::vector<S2CellId> *OUTPUT {std::vector<S2CellId> *interior};
 %apply std::vector<S2CellId> *OUTPUT {std::vector<S2CellId> *output};
 
+%typemap(in) absl::string_view {
+  if (PyUnicode_Check($input)) {
+    $1 = absl::string_view(PyUnicode_AsUTF8($input));
+  } else {
+    SWIG_exception(SWIG_TypeError, "string expected");
```

Comment:
a string_view may also be pointing at non-utf-8 data, but I guess that's not happening. How do we add that if we need to?

Reply:
The string in this case is coming from python though, and [SWIG's default behaviour is to map std::string ↔︎ String](http://www.swig.org/Doc3.0/Python.html#Python_nn77) in Py3.

Could use `SWIG_PYTHON_STRICT_BYTE_CHAR` and force all string encoding to happen in Python. Not sure if that solves a real use-case though?

Reply:
I guess I could easily convert Python Bytes without encoding to `std::string`, but not sure what would happen on the way back out, considering SWIG's default behaviour.